BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -43,5 +44,29 @@ public async Task<IActionResult> Register([FromBody] RegisterRequest model)
             return Ok(responseId);
 
         }
+
+        [HttpGet]
+        [Route("profiles/{userId:int}")]
```

Comment:
I missed that in Andrėjus' PR. We should not use `int` as Profile ID. `int` is for database-generated id, which is incremental and easily brute forceable. We should use "external" ID which is simply random GUID string.

Reply:
Updated with new commit